REFERENCE.md

```diff
@@ -7893,14 +7906,43 @@ Alias of
 Variant[Stdlib::Fqdn, Stdlib::Compat::Ip_address]
 ```
 
+### <a name="stdlibhttpmethod"></a>`Stdlib::Http::Method`
+
+Valid HTTP method verbs
+
+* **See also**
+  * https://www.iana.org/assignments/http-methods/http-methods.xhtml
+
+Alias of
+
+```puppet
+Enum['ACL', 'BASELINE-CONTROL', 'BIND', 'CHECKIN', 'CHECKOUT', 'CONNECT', 'COPY', 'DELETE', 'GET', 'HEAD', 'LABEL', 'LINK', 'LOCK', 'MERGE', 'MKACTIVITY', 'MKCALENDAR', 'MKCOL', 'MKREDIRECTREF', 'MKWORKSPACE', 'MOVE', 'OPTIONS', 'ORDERPATCH', 'PATCH', 'POST', 'PRI', 'PROPFIND', 'PROPPATCH', 'PUT', 'REBIND', 'REPORT', 'SEARCH', 'TRACE', 'UNBIND', 'UNCHECKOUT', 'UNLINK', 'UNLOCK', 'UPDATE', 'UPDATEREDIRECTREF', 'VERSION-CONTROL']
+```
+
+### <a name="stdlibhttpstatus"></a>`Stdlib::Http::Status`
```

Comment:
This is "funny": strings generates the same anchor for `Stdlib::Http::Status` as `Stdlib::HttpStatus`.

Reply:
hmm not sure we can do much about it here, i have create an [issue](https://github.com/puppetlabs/puppet-strings/issues/341) in puppet strings

Reply:
I don't think there's much we can do here other than what you did. I think the good news is that it generates the one we do want first so it's likely browsers will use that anchor.

I wonder why they do this since they could use `-` (which is invalid in a Puppet type, so should work). According to https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id that should be fine.

TIL: the `name` attribute is actually deprecated according to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a and they recommend `id` instead.

Reply:
look is like this has already been [reported](https://github.com/puppetlabs/puppet-strings/issues/300) and [fixed](https://github.com/puppetlabs/puppet-strings/pull/303).  seems that bolt is locking puppet-strings to ~> 2.3

Reply:
ok so i see the fix to bolt is included in 3.27 which was released 17hours ago, however its not yet on rubygems.  As such i think we can probably wait a day or two for bolt to get released then `bundle update` and regenerate

Reply:
i have regenerated references.md in #1301 once that's merged ill rebase

Reply:
ok updated